konfera/settings.py

```diff
@@ -70,3 +70,14 @@
 How long should we wait to notify about missing payment
 """
 UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY = getattr(settings, 'UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY', 5)
+
+"""
+Absolute url base with protocol
```

Comment:
please add a note it always have to have end without (once we use reverse) trailing slash (/) so it is explicit.